lib/circuitry/publisher.rb

```diff
@@ -60,8 +61,8 @@ def self.default_async_strategy
 
     protected
 
-    def publish_message(topic_name, message)
-      middleware.invoke(topic_name, message) do
+    def publish_message(topic_name, message, message_attributes)
```

Comment:
Should we also have a default value for `message_attributes`? Its a protected method, not sure if its being called anywhere